aea/multiplexer.py

```diff
@@ -347,7 +347,7 @@ async def _receiving_loop(self) -> None:
 
         while self.connection_status.is_connected and len(task_to_connection) > 0:
             try:
-                logger.debug("Waiting for incoming envelopes...")
+                # logger.debug("Waiting for incoming envelopes...")
```

Comment:
Problematic debug statement which spams the log..

Reply:
ok, remove?